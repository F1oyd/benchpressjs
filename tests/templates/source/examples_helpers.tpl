<!-- IF function.isEvenLegacy, ten -->
{function.caps, lorem}
<!-- END -->

{{{ if isEven(ten) }}}
    {caps(lorem)}
{{{ end }}}



The first five letters are: {join(", ", "a", "b", "c", "d", "e")}

{{{ if !isEven(eleven) }}}
It's odd.
{{{ end }}}
