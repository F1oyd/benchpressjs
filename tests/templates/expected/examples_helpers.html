LOREM IPSUM DOLAR SIT AMET

LOREM IPSUM DOLAR SIT AMET



The first five letters are: a, b, c, d, e

It's odd.